mathext/hyp2f1_test.go

```diff
@@ -0,0 +1,230 @@
+// Tests in this file are taken from R's hypergeo package.
```

Comment:
This has license implications, as above. I would move this down to the tests var though.